test/util.dart

```diff
@@ -107,7 +107,8 @@ class RefTestCaseCollection<T> {
     final calls = [];
     return RefTestCase(
       name: name,
-      ref: (value) => calls.add(value),
+      // ignore: unnecessary_lambdas, avoid_types_on_closure_parameters
+      ref: (dynamic value) => calls.add,
```

Comment:
Sorry, this was from a bad suggestion from me (and is causing failing tests)
```suggestion
      ref: (dynamic value) => calls.add(value),
```